api/src/lib.rs

```diff
@@ -373,40 +373,71 @@ pub async fn fetch_github_data(
     Ok(response.json().await?)
 }
 
+/// Modify the body of a pull request to display links, images, and code blocks in a format that
+/// can be displayed in a code block without being rendered as a link.
+/// This is useful for displaying the body of a pull request without the links referencing the
+/// actual pull request.
+/// For example, a link to a pull request will be displayed as `#123` instead of a clickable link.
+/// Images and code blocks will be converted to markdown format.
 pub fn modify_pull_request_body(args: Option<&str>) -> String {
     if let Some(args) = args {
-        // Regex for inline code blocks, GitHub links, and markdown links
+        // Regex for finding and replacing code blocks, links, and images
         let code_block_re = Regex::new(r"`([^`]*)`").unwrap();
```

Comment:
I think we should use a more graceful error handling when compiling the regular expressions, which can lead to panic if the regular expression patterns are invalid.

Can we consider using expect() to provide a more informative error message

Reply:
addressed https://github.com/bitcoin-dev-project/fresh-eyes/pull/22/commits/61e39d1fc3991c17484a6cac261abc0595058dd4